### DISCUSSION TITLE
CodeAssist的代码不能直接兼容AIDE

### DISCUSSION THREAD

**Discussion by Alan3344** [tyron12233/CodeAssist#407] [Ideas]
![Screenshot_20220328-100708](https://user-images.githubusercontent.com/60773912/160314923-9728b989-bea3-4fca-85b8-6b6eb2925c28.jpg)
![Screenshot_20220328-101254](https://user-images.githubusercontent.com/60773912/160314971-4d811960-9c39-437c-ad21-031a0e27d1e9.jpg)

**Comment by dingyi222666** [comment]
CodeAssist里面的依赖库解析是自己管理的，不是直接放在gradle脚本里面的

**Reply by NewBeeqvq** [reply]
牛批
